### DISCUSSION TITLE
disable entity fire damage overlay

### DISCUSSION THREAD

**Discussion by ghost** [maruohon/tweakeroo#390] [Ideas]
a "yeet" option for tweakeroo that disables the large flame texture used to indicate if an entity is in a block that makes them burn.

**Comment by maruohon** [comment]
Do you mean for the player in first person perspective, or for all/other entities?

**Reply by ghost** [reply]
for all entities. the big flame thats added above/over entities when on fire.

**Reply by tmuaprep** [reply]
Can you do it in first person perspective?
